### DISCUSSION TITLE
Miravist GA4 add_cart_event not fire

### DISCUSSION THREAD

**Discussion by phpDevChandra** [breezefront/community#154] [5. General]
Hi All,

I am using latest breeze module 2.7.0 and miravist GA4 module but add_cart_event not trigger in tage manager.
I have disabled all 3rd party and custom module and still same.
I checked older breeze module 2.5.5 but same issue
When changed theme in LUMA then its working.
Can you please suggest / update what could be cause for  ? OR any issue with latest breeze module or else.

Thanks.

**Comment by vovayatsyuk** [comment]
Are you using mirasvit-googletagmanager? You need to install the integration module: https://breezefront.com/extensions#mirasvit-googletagmanager

If it's a different module - we don't have integration with it.
